src/codegeneration/ObjectLiteralTransformer.js

```diff
@@ -54,7 +54,19 @@ class FindAdvancedProperty extends FindVisitor {
     if (isProtoName(tree.name))
       this.protoExpression = tree.value;
     else
-      super.visitPropertyNameAssignment(tree);
+      return super.visitAny(tree.name);
+  }
+
+  visitPropertyMethodAssignment(tree) {
+    return super.visitAny(tree.name);
```

Comment:
Just do:

```js
this.visitAny(tree.name);
```

no need to go through super here.

This could also use a comment.